packages/iTwinUI-react/src/core/Table/cells/EditableCell.tsx

```diff
@@ -80,6 +84,9 @@ export const EditableCell = <T extends Record<string, unknown>>(
           onCellEdit(cellProps.column.id, value, cellProps.row.original);
         }
         props.onBlur?.(e);
+        // Prevents error when text is cleared.
+        // New key makes React to reattach with the DOM so it won't complain about deleted text node.
+        setKey(getRandomValue(10));
```

Comment:
So I know React treats new key as a new node but why does this fix the error? Any reading material on this?

Reply:
I found about this here: https://stackoverflow.com/a/67686478/18045303